topic-configuration/README.adoc

```diff
@@ -696,5 +698,124 @@ h|Kafka property name
+
+To use a schema, a client application can directly publish a new schema to a {registry} instance itself, or use one that is already created there. In either case, to establish an association between the schema and a Kafka topic, client application code is typically configured to use a strategy whereby the schema ID must use the name of the topic. Specifically, to match an existing topic, a value or key schema must use a naming format of `_<topic-name>_-value` or `_<topic-name>_-key`.
+
+However, to identify the associations between schemas in {registry} and Kafka topics in {product}, it might not always be convenient for you or others in your organization to inspect client application code. Instead, to quickly identify these associations, you can use the {product} web console.
+
+For a given Kafka topic, you can use the console to check {registry} instances for value or key schemas registered to those instances that match the name of the topic. If you don't have access to any {registry} instances, or don't have value or key schemas registered to your instances that match your topic, the console provides links to {registry}, where you can create these. The console also shows you the naming format you need to use when creating a new value or key schema, so that it matches the topic.
+
+[id="proc-checking-topic-for-existing-schema-matches_{context}"]
+== Checking a topic for existing schema matches
+
+[role="_abstract"]
+The following procedure shows how to use the {product} web console to select a Kafka topic and then check an existing {registry} instance for value or key schemas that have IDs that match the name of the topic.
+
+Alternatively, to learn how to create a _new_ {registry} instance with a value or key schema that matches a topic, see {base-url}{topic-config-url}#proc-creating-registry-instance-and-schema_{context}[Creating a new {registry} instance and matching schema for a topic].
+
+.Prerequisites
+* You're logged in to the {product} web console.
+* You've created a Kafka instance with at least one topic in {product}. To learn how to do this, see {base-url}{getting-started-url}[Getting started with {product-long}^].
+* You understand how to create a {registry} instance and upload a schema to be used by client applications. To learn how to do this, see {base-url}getting-started-service-registry/README.adoc[Getting started with {registry}^].
+//TO DO:  Add attribute for SR getting started guide
+* You have access to at least one instance in {registry} that you can check for schemas that match your topic.
+
+.Procedure
+. In the {product} web console, go to *Streams for Apache Kafka* > *Kafka Instances*. Click the name of the Kafka instance that contains the topic that you want to check for matching schemas in {registry}.
```

Comment:
Suggest changing "check for matching schemas" to "check for associated schemas"

Reply:
Thanks, @smccarthy-ie. Yeah, I was probably overthinking things and ended up with awkward marriage of the two terms! I wanted to initially use a slightly loose term to represent the coupling between schemas and topics, until we could properly define what a _match_ means. I chose _association_ for that purpose. The paragraph on line 708 was intended to make a connection between the two terms, so that we could use _match_ thereafter.

Anyway, that's enough of an explanation. Looking again, I think we can eliminate these inconsistencies. I will use _match_ where I can.